src/Microsoft.TemplateSearch.ScraperOutputComparison/TemplateEngineHostHelper.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
Good chance to add `#nullable enable`

Reply:
That file was moved :)  - I will add but I think we can do it in tools in one go when time allows.